packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -234,6 +235,40 @@ exports.mapTagsCategoriesToTaxonomies = entities =>
     return e
   })
 
+exports.searchReplaceContentUrls = async function ({ entities, searchReplace }) {
```

Comment:
these functions don't need to be async — unless you're doing file or network IO this should be a normal synchronous function.

Reply:
Your right, I am normally used to writing non blocking code for node.. I'll make this change 👍